cellprofiler/gui/help/menu.py

```diff
@@ -185,10 +184,17 @@ def __on_help_developers_guide():
         import webbrowser
         webbrowser.open("https://github.com/CellProfiler/CellProfiler/wiki")
 
-    # @staticmethod
-    # def __on_help_online_manual(event):
-    #     import webbrowser
-    #     webbrowser.open("http://d1zymp9ayga15t.cloudfront.net/CPmanual/index.html")
+    @staticmethod
+    def __on_help_online_manual(event):
+        import requests
```

Comment:
You could bump these imports to the module space since `requests` is imported elsewhere and `webbrowser` is part of the Python standard library.